widgetastic_manageiq/__init__.py

```diff
@@ -4725,3 +4727,12 @@ def select_plan(self, plan_name):
         """Find item by text and click to view details."""
         el = self._get_card_element(plan_name)
         self.browser.click(self.TITLE_LOCATOR, parent=el)
+
+    def get_progress_percent(self, plan_name):
+        """Returns datastore and vm progress bar percentage"""
+        el = self._get_card_element(plan_name)
+        _desc = [desc.text.lower()
+                 for desc in self.browser.elements(self.PROGRESS_DESCRIPTION, parent=el)]
+        _val = [float(div.get_attribute("style").split(": ")[1].split("%")[0])
```

Comment:
Please include in-line comments about what you're expecting to see in the DOM here, and how you're expecting to process it.

Reply:
@mshriver  added comment like `# extracting style attribute values and returning it as percentage xx.xx%`, hope it will be helpful.